### DISCUSSION TITLE
Compass interaction…

### DISCUSSION THREAD

**Discussion by Cap-Am** [Kemmey/Kemmey-TeslaWatch-Public#515] [Q&A]
One of the reasons I got the app was a nice YouTube vid showing it off… especially the compass that would point in the actual direct on your car. Finding my car at a huge work parking lot that is always full would be helpful. My compass just shows what cardinal direction it is from me, but doesn’t help as it doesn’t account for current direction of the watch without me having to switch between the apps. I couldn’t find a setting to change this- could that be an option again?

**Comment by Kemmey** [comment]
https://user-images.githubusercontent.com/4688084/201923740-3c053b01-6d5d-4dc5-8d9d-587c71ad8be4.MOV

It can and does?

What Watch do you have? Does the hardware have a compass physically? Older watches don't.  

Have you granted location permissions when the app asked for them? If not, grant that from the iOS or watchOS settings app.

**Reply by Cap-Am** [reply]
I’m glad it’s still available, now I have to figure out how to get mine working. I have the Apple Watch 6, and it does have a compass. I’ll see if I can send a video… my app compass does not turn like that. It does show compass rose and points (most times) in accurate direction w accurate distance. Sometimes, however it will have a very wide guess at the direction and the distance is 6,600 plus miles…

**Reply by Cap-Am** [reply]
![3FC1DDCE-A1C5-4699-A4F3-D25B60DE2225](https://user-images.githubusercontent.com/118274890/201925453-af51689c-8670-4aaa-9c3f-9e4d741f7bd4.jpeg)

**Reply by Cap-Am** [reply]
It’s unusual that it is 6k miles and a narrow locator swath. 🤷‍♂️

**Comment by Cap-Am** [comment]
Either I have added the video 3 times already or it isn’t letting me do it…

**Reply by Cap-Am** [reply]
![8B7B4E51-7187-4C48-B067-505B3E833357](https://user-images.githubusercontent.com/118274890/201927248-067083f1-db26-4600-a15e-f028d72002db.jpeg)
![AC7FEF9F-4587-4CE0-8BE1-37DD18D8B648](https://user-images.githubusercontent.com/118274890/201927254-4626fc8e-8f43-4871-b329-13677a3b0380.jpeg)
![1027AC72-2CF7-4108-A3DF-E559C5D682B8](https://user-images.githubusercontent.com/118274890/201927260-97845518-6a09-4159-ba32-db2ae952084b.jpeg)
Spinning in the same room (see carpet angles)

**Reply by Cap-Am** [reply]
Also, location permission is on.

**Comment by Kemmey** [comment]
![image](https://user-images.githubusercontent.com/4688084/201928050-b494d875-2844-4a12-8f29-82c979071ebd.jpeg)
![image](https://user-images.githubusercontent.com/4688084/201928072-fd2bea3b-5958-47ef-9b6e-f52f0210c0b9.jpeg)

Please check location permissions on your phone - ensure you verify permissions for the watch app with the round icon - not the phone app with the square icon.

**Reply by Cap-Am** [reply]
Permission is enabled on my only watch app… that happens to be rectangular… so that may be the problem? I checked the App Store for a round version, but don’t see that option.

**Reply by Cap-Am** [reply]
Well, I think I found the problem. In Privacy, Location services, system services…
Unless “compass calibration” is on, it will stand still as described. Sorry to take up so much time, but maybe it will help with any other people with similar problems. 
thank you very much for your help!
Donald Kingsley

**Comment by Zhou-Young** [comment]
好的，已收到～
